### ISSUE TITLE
use question

### ISSUE THREAD

**Issue by dongbo12300** [toji/gl-matrix#262]
how can i use glmatrix in typescripte project？Looking forward to your reply.

**Comment by Kaesebrot84** [comment]
Hi,
first make sure that you have the `@types/gl-matrix` installed. You can install them via:

`npm install --save @types/gl-matrix`

See the [npm-page](https://www.npmjs.com/package/@types/gl-matrix) for further information on types.

Now you should be able to load and use `gl-matrix ` in your TypeScript file like this:
```javascript
import * as glMatrix from 'gl-matrix';
// ...
let matrix = glMatrix.mat4.create();
```

**Comment by 8Observer8** [comment]
I created a little game in TS and WebGL 1.0. You can play here: https://8observer8.github.io/WebGL/Bitballoon/

You can see how I used glMatrix from TS here: https://github.com/8Observer8/Bitballoon3D

**Comment by Kaesebrot84** [comment]
@toji @stefnotch I think this one can be closed, right? Also see #216

**Comment by stefnotch** [comment]
@Kaesebrot84 So, https://github.com/DefinitelyTyped is going to take care of everything and I don't have to care about anything? 

Also, feel free to only ping me for "unimportant" stuff, because toji is apparently rather busy. 😉
